int64 ch
float64 voltage
float64 interval
float64 timestamp